internal/controller/cloudsyncmgr/cloudsync.go

```diff
@@ -29,6 +29,7 @@ import (
 
 const (
 	logPrefix = "[cloudsyncmgr]"
+	mqttPort  = 8883
```

Comment:
It would be better to set 8883 during `secure` mode. Otherwise, the number must be set to 1883.

Reply:
Seems better option. Done!